pages/linux/grub2-mkpasswd-pbkdf2.md

```diff
@@ -0,0 +1,8 @@
+# grub2-mkpasswd-pbkdf2
+
+> Set up a password to prevent unauthorized users from accessing the GRUB command line, modifying kernel command-line arguments, or booting non-default OSTree deployments.
```

Comment:
This command does not actually set anything up. It only prints the hashed password to standard output. I think the current description is kind of misleading.
```suggestion
> Generate a hashed password for GRUB.
```